src/components/organisms/tables/SubOrdersTable/SubOrdersTable.tsx

```diff
@@ -107,7 +107,12 @@ const SubOrdersTable: FC = () => {
   const onSubmit: SubmitHandler<FormValues> = useCallback(
     (payload) => {
       handleFilterChange({
```

Comment:
handleFilterChange({
         ...payload,
           only_show_personal_projects: payload?.only_show_personal_projects ?? 0
       })
       
Might be easier on the eye